### PR TITLE
Added a global flag to disable accessibility feature of larger text

### PR DESCRIPTION
**Pull request by mohaider**
## PR Type
Added a global flag  `FeatureConfiguration.Font.IgnoreTextScaleFactor'. Setting this true to false would ignore text scaling based on the accessibility function of devices.  

Fixed spacing in UIFontHelper and FontHelper 

## What is the current behavior?
Currently there is no way of ignoring accessibility feature of larger text. This change introduces a way of managing this function.

## What is the new behavior?
 etting this true to false would ignore text scaling based on the accessibility function of devices.  

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Tested code with current [supported SDKs](../README.md#supported)
- [x] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
- [ ] Tests for the changes have been added (for bug fixes / features) (if applicable)
- [x] Contains **NO** breaking changes
- [x] Updated the [Release Notes](https://github.com/nventive/Uno/tree/master/doc/ReleaseNotes)
- [x] Associated with an issue (GitHub or internal)  

Internal Issue (If applicable):
https://nventive.visualstudio.com/Umbrella/_workitems/edit/35673->

### CODE REVIEW

File: doc/ReleaseNotes/_ReleaseNotes.md [hunk 1]
```diff
@@ -44,6 +44,7 @@
 * Add `FeatureConfiguration.FrameworkElement.AndroidUseManagedLoadedUnloaded` to control the native or managed propagation performance of Loaded/Unloaded events through the visual tree
 * Raise Application.UnhandledException event on failed navigation
 * Adjusts the `Microsoft.NETCore.UniversalWindowsPlatform` version in the UWP head template to avoid assembly loading issues when using the Uno library template in the sample solution.
+* Added support to disable accessibility feature of larger text on iOS and Android by adjusting the FeatureConfiguration.Font.IgnoreTextScaleFactor flag. Please note that Apple [recommends to keep text sizes dynamic](https://developer.apple.com/videos/play/wwdc2017/245) for a variety of reasons and to allow users to adjust their text size preferences.
```
Comment (MatFillion) [inline doc/ReleaseNotes/_ReleaseNotes.md]: Also add it to FeatureFlags.md and in Working-With-Accessibility

File: doc/articles/working-with-accessibility.md [hunk 2]
```diff
@@ -35,6 +35,15 @@ FeatureConfiguration.AutomationPeer.UseSimpleAccessibility = true;
...
+##Disabling accessibility text scaling
+
+You have the option to disable accessibility text scaling of iOS and Android devices but Apple [recommends to keep text sizes dynamic](https://developer.apple.com/videos/play/wwdc2017/245) 
+
+Here's how to disable it
+```csharp
+// App's constructor (App.xaml.cs) 
+Uno.UI.FeatureConfiguration.Font.IgnoreTextScaleFactor= true; 
+```
 We highly recommend using this mode, as iOS still won't let you focus nested accessible elements even if you don't (see known issues).
```
Comment (MatFillion) [inline doc/articles/working-with-accessibility.md]: This last phrase is now in the wrong section no? "We Highly recommend using this more...", it should be above ##Disabling accessibility text scaling

Comment (mohaider) [reply doc/articles/working-with-accessibility.md]: should be resolved now.

**Comment by jeromelaban** [comment]
Did you check for this: https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.control.istextscalefactorenabled ?

**Pull request merged**